### ISSUE TITLE
(click) events not working after loopCreate is called

### ISSUE THREAD

**Issue by sanctus671** [zefoy/ngx-swiper-wrapper#236]
I've been trying to get a slider working which loops and has clickable elements. My config is as follows:

`this.sliderConfig = {slidesPerView: 3, loop:true, loopedSlides: 6,   spaceBetween: 15, navigation: {nextEl: '.swiper-button-next',prevEl: '.swiper-button-prev',}};`

With this alone, the loop does not work. I found that you have to call LoopCreate() explicitly after initialization to get it to work, like so:

```
    ngAfterViewInit() {
        const swipers = document.querySelectorAll('.swiper-container');   
        swipers.forEach((swiper) => {
            swiper['swiper'].loopCreate();  
         })
    }
```

The loops works, but now none of the (click) events on the slider items work until you swipe on the slider. At that point I assume the click events are added in by swiper.js.

This is a problem if a user just clicks on one of the first slides without swiping or navigate to other slides first. The (click) won't be called.

Any ideas how I can solve this?

**Comment by sconix** [comment]
Have you tried calling update after the DOM is updated, many of the Swiper features need update call for it to register stuff after Angular has finished updating the DOM.

**Comment by sanctus671** [comment]
Yes, have tried that. When I do that, the slider constantly loops back to the first slide. As in, if I try to navigate to the second slide, it slides to the second slide, but then all the slides get updated and now the second slide is the first slide again (sorry if that is a bit confusing). Something glitchy with the loop I assume. The only way I could get the loop to work correctly is by putting the function in ngAfterViewInit. However this gives the (click) problem as described earlier.

**Comment by sconix** [comment]
The loop is bit tricky option since it modifies DOM so things are happening without Angular knowing and the other way around. Would be nice if Swiper would provide better support for dynamic changes. So I have no more ideas, never needed the loop option so have not had to battle with it.
